packages/server/src/main.ts

```diff
@@ -1,8 +1,12 @@
 import { NestFactory } from '@nestjs/core';
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  const config = new DocumentBuilder().setTitle('Email Template').setDescription('The Email Template API description').setVersion('1.0').addTag('preview').build();
```

Comment:
This document is for the whole service
```suggestion
  const config = new DocumentBuilder().setTitle('Notification Service').setDescription('The notifications microservice provides a common abstraction layer on top of the various notification types and actual sending.').setVersion('1.0').build();
```